db/schema.rb

```diff
@@ -519,6 +519,7 @@
     t.integer "prevten"
     t.integer "mortgageused"
     t.integer "wchair"
+    t.integer "income2_value_check"
```

Comment:
i don't recommend checking in the schema file if you haven't actually made any schema changes

Reply:
good point